roles/addons/prometheus_server/molecule/default/prepare.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Prepare
+  hosts: all
+  tasks:
+    - name: Install needed packages
+      package:
+        name: "{{ item }}"
+        state: present
+      when: ansible_facts.os_family == "RedHat"
+      loop:
+        - firewalld
+        - httpd
```

Comment:
Why do you need to install httpd during the verify step?

Reply:
no need